socorro/external/postgresql/crashstorage.py

```diff
@@ -25,20 +17,16 @@
     class_converter
 )
 from socorro.external.postgresql.connection_context import ConnectionContext
-from socorro.lib.datetimeutil import uuid_to_date, JsonDTEncoder
+from socorro.lib.datetimeutil import uuid_to_date
 from socorro.external.postgresql.dbapi2_util import (
     SQLDidNotReturnSingleValue,
     single_value_sql,
     execute_no_results
 )
 
 
-class PostgreSQLBasicCrashStorage(CrashStorageBase):
-    """this implementation of crashstorage saves processed crashes to
-    an instance of Postgresql.  It only saves certain key values to the
-    partitioned reports table, therefore it is not a source for fetching
-    complete processed reports and doesn't not implement any of the 'get'
-    methods."""
+class PostgreSQLCrashStorage(CrashStorageBase):
```

Comment:
I figured it was easier to squash these two classes than keep them separate. We have no need of having two separate classes and it doesn't help us at all--it's just more maintenance.

Squashing them was pretty straight-forward: couple of name changes, move a few functions, remove a bunch of tests, and done.